cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -825,6 +831,11 @@ void ExportedStubFunctions::AddMappings(Engine* engine) const {
   engine->AddGlobalMappingForFunc("gdv_fn_random_with_seed", types->double_type(), args,
                                   reinterpret_cast<void*>(gdv_fn_random_with_seed));
 
+  args = {types->i64_type(), types->i64_type(), types->i1_type()};
+  engine->AddGlobalMappingForFunc("gdv_fn_random_with_seed_int64", types->double_type(),
```

Comment:
Why not always mandate int64 instead of defining two different functions?

Reply:
To make the random function support both int64 and int32 as the seed, it requires 2 function signatures with different parameter types.

Reply:
That doesn't answer the question.

Reply:
Do you mean remove the int32 one and only support int64? I see the gdv registry is quite restrict. Say the "add" function, it takes "add_int8_int8", "add_int16_int16", "add_int32_int32" ... to support different param types.

Reply:
Yes, I mean that.

Reply:
Hmmm, will that bring regression?